packages/blockchain-wallet-v4-frontend/src/services/WhatsNewService/WhatsNewContent/index.js

```diff
@@ -20,18 +20,23 @@ const Announcements = [
 ]
 
 export const filterAnnouncements = (lastViewed, userCountry, userKycState) => {
-  const isOnRestrictedCountryList = restrictedCountryList => restrictedCountryList.indexOf(userCountry) > -1
-  const isOnRestrictedKycList = restrictedKycList => restrictedKycList.indexOf(userKycState) > -1
+  const isOnRestrictedCountryList = contains(userCountry)
+  const isOnRestrictedKycList = contains(userKycState)
 
-  const isRestricted = announcement => (
+  const isRestricted = announcement =>
     isOnRestrictedCountryList(prop('restrictByCountry', announcement)) ||
     isOnRestrictedKycList(prop('restrictByUserKyc', announcement))
-  )
 
-  const filteredAnnouncements = Announcements.filter(announcement => {
-    if (!isRestricted(announcement)) {
-      return moment(announcement.date).isBetween(moment(lastViewed), moment())
-    }
-  })
-  return filteredAnnouncements
+  const isAvailableToView = (start, expires) =>
+    moment(start).isBetween(
+      moment(lastViewed).subtract(expires, 'days'),
+      moment()
+    )
+
+  return Announcements.map(announcement => ({
+    content: prop('content', announcement),
+    restricted: isRestricted(announcement),
+    display: isAvailableToView(lastViewed, 3),
+    alert: isAvailableToView(announcement.date, 0)
+  }))
```

Comment:
The new logic is:

1. Display the content if the feature has been viewed more than 3 days ago
2. Display the alert (number above icon) if the icon has been clicked after the announcement date